### ISSUE TITLE
unable to get Ambassador Cloud preferred agent image

### ISSUE THREAD

**Issue by wnke** [telepresenceio/telepresence#2625]
**Describe the bug**
`telepresence intercept ` fails with a timeout. And no agent is ever added to the pod.

Additionally looking into the traffic-manager logs:
```
2022-06-01 10:11:38.1113 warning httpd/conn=127.0.0.1:8081 : unable to get Ambassador Cloud preferred agent image: rpc error: code = Unauthenticated desc = transport: per-RPC creds failed due to error: no apikey has been provided by a client : session_id="1b49b874-94a2-4d2a-b55f-ed15b6d8e046"
2022-06-01 10:11:40.8706 warning systema/client : error talking to System A: rpc error: code = Unauthenticated desc = transport: per-RPC creds failed due to error: no apikey has been provided by a client; retrying in 5s
```

This started happening since updating to 2.6.4, previously using 2.5.8. And seems to be related with
``` No default image in the Helm chart ``` on release 2.6.0 and the fact that the traffic-manager is unable to get the image to use.

** reproduce the error **

`telepresence intercept tasse --namespace=staging --port=8099:8099
`
**Expected behavior**
Intercept agent is deployed along side the the tasse pod

**Versions (please complete the following information):**
```
$ telepresence version
Client: v2.6.4 (api v3)
Root Daemon: v2.6.4 (api v3)
User Daemon: v2.6.4 (api v3)
```

Is there any way to set a default image via the telepresence command?

**Comment by thallgren** [comment]
The problem i very likely fixed in 2.6.5.

To answer your question. There are ways to set the default image. Which way to use depends on how you install the traffic-manager into your cluster. If you rely on the auto-install (where the Telepresence installs it the first time you connect), then you can specify the image in your `config.yml` as:
```
images:
  agentImage: ambassador-telepresence-agent:1.12.3
```
and if you are using Helm to install, then you can provide the value
```
agentInjector:
  agentImage:
      name: ambassador-telepresence-agent
      tag: 1.12.3
```

**Comment by thallgren** [comment]
Closing this a fixed. Please reopen if you still experience these problems.
